README.md

```diff
@@ -130,41 +132,47 @@ You should see a JSON response like that below:
   "predictions": [
     {
       "mask": [
-        400,
-        2,
-        463,
-        3,
-        527,
-        3,
-        590,
+        3507,
```

Comment:
Just a question - why has the output for `mask` here been changed?

Reply:
I did not find the image I used at the very beginning for generating the mask output, so I use another image this time. The `example.png` in `Assets` has a very long output and is only used for the demo notebook.